source/text.tex

```diff
@@ -12238,8 +12238,8 @@
 constructs an end-of-sequence iterator object,
 which is the only legitimate iterator to be used for the end
```

Comment:
This is somewhat misleading since we added comparison with `default_sentinel_t` although that's not actually an iterator so it's technically true, just misleading.